src/dataflowd/src/bin/dataflowd.rs

```diff
@@ -92,6 +92,8 @@ struct Args {
     storage_addr: String,
     #[clap(long, default_value = "0")]
     storage_workers: usize,
+    #[clap(long)]
+    linger: bool,
```

Comment:
Just making it up, but I initially imagined this being an `Option<Duration>`. I'm not sure if we have a preferred mode of operation yet, and happy to wait for that to evolve a bit before making requests!

Reply:
One use case where I think we want to have it lingering indefinitely is testdrive. All processes are created ahead of time, and it would be strange if they disappeared through a run. Now, we could also have some integration with how pods are started, but I rather have something that works now can easily be changed.